namespace cpp entity
namespace py ccentity.cooling
namespace as3 com.dreamfire.sanguo.data.definition.entity

/////////////////////////////////
// 冷却信息
struct CoolingFields {
  1 : required i32 id_;                   // 冷却ID
  2 : required string elapse_time_;       // 逝去时间
}

